lib/board.rb

```diff
@@ -0,0 +1,57 @@
+class Board
+  attr_accessor :cells
+
+  def reset!
+    @cells.clear
+    @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+  end
```

Comment:
having `@cells.clear` is a bit redundant -- the second line reassigns `@cells` to the array you want, so it doesn't matter what its previous value was.